benchmark/profile_generation.py

```diff
@@ -165,15 +169,23 @@ def profile_throughput(model_path: str, concurrency: int, input_seqlen: int,
                                  3)
     token_latency_ave = np.round(np.mean(np.sum(token_latency_stats, axis=1)),
                                  3)
-    # sort token_latency without the first token's latency
-    sorted_token_latency = np.sort(token_latency_stats[:, 1:].flatten())
-    percentiles = [
-        np.round(
-            sorted_token_latency[int(percent * len(sorted_token_latency))], 3)
-        for percent in [0.5, 0.75, 0.95, 0.99]
-    ]
-
-    throughput = np.round(token_latency_stats.size / elapsed_time, 2)
+    if output_seqlen > 1:
```

Comment:
Do we need to support this? Prompts with `output_len<4` were filtered.

Reply:
It is profile_generation with dummy token_ids, not profile_throughput with ShareGPT data.
We don't do any filters in profile_generation